test/powershell/Modules/Microsoft.WSMan.Management/ConfigProvider.Tests.ps1

```diff
@@ -0,0 +1,436 @@
+            try {
+                Stop-Service WinRM
+                Get-ChildItem wsman:\localhost -Force | Should Not BeNullOrEmpty
+                (Get-Service WinRM).Status | Should Be 'Running'
+            }
+            finally {
+                if ((Get-Service WinRM).Status -eq 'stopped') {
+                    Start-Service WinRM
+                }
+            }
+        }
+    }
+
+    Context "Get-Item tests" {
+
+        It "Plugin has correct properties" {
+            $plugin = Get-Item $pluginPath
+            Test-Plugin -Plugin $plugin
+        }
+
+        It "Plugin InitializationParameters are correct" {
+            $initializationParameters = Get-ChildItem $pluginPath\InitializationParameters
+            $initializationParameters.Count | Should Be (,$pluginXml.PluginConfiguration.InitializationParameters.Param).Count
+            foreach ($initializationParameter in $initializationParameters) {
+                $initializationParameter.Value | Should Be ($pluginXml.PluginConfiguration.InitializationParameters | Where-Object { $_.Param.Name -eq $initializationParameter.Name }).Param.Value
+                (Get-Item $pluginPath\InitializationParameters\$($initializationParameter.Name)).Value | Should Be ($pluginXml.PluginConfiguration.InitializationParameters | Where-Object { $_.Param.Name -eq $initializationParameter.Name }).Param.Value
+            }
+        }
+
+        It "Plugin Quotas are correct" {
+            $quotas = Get-ChildItem $pluginPath\Quotas
+            $quotas.Count | Should Be ($pluginXml.PluginConfiguration.Quotas | Get-Member -Type Properties).Count
+            foreach ($quota in $quotas) {
+                $quota.Value | Should Be $pluginXml.PluginConfiguration.Quotas.$($quota.Name)
+            }
+        }
+
+        It "Plugin Resources are correct" {
+            $resources = Get-ChildItem $pluginPath\Resources
+            $resources.Count | Should Be (,$pluginXml.PluginConfiguration.Resources.Resource).Count
+        }
+
+        It "Plugin Security Resource is correct" {
+            (,$pluginXml.PluginConfiguration.Resources.Resource).Count | Should Be 1 # by default only Security resource should be there
+            $resource = Get-ChildItem "$pluginPath\Resources" | Select-Object -First 1
+            $resourceUri = Get-Item "$($resource.PSPath)\ResourceUri"
+            $resourceUri.Value | Should Be "http://schemas.microsoft.com/powershell/microsoft.powershell"
+            $securityContainer = Get-ChildItem "$pluginPath\Resources\$($resource.Name)\Security" | Select-Object -First 1
+            $securityProperties = Get-ChildItem $securityContainer.PSPath
+            $skippedAttributes = @("ParentResourceUri","xmlns") # these are added by the WSMan Config Provider but not in the original xml
+            # ParentResourceUri is added by the Config Provider, xmlns existing seems to be dependent on WinRM which is inconsistent
```

Comment:
Please place a comment on new line.

Reply:
I think the request here was to move the comment on line 115 to a new line, not insert a newline character after it.

This isn't a blocking issue for the PR. I'm just clarifying intent.